sharding-transaction/sharding-transaction-base/sharding-transaction-base-seata-at/src/main/java/org/apache/shardingsphere/transaction/base/seata/at/SeataATShardingTransactionManager.java

```diff
@@ -47,34 +47,56 @@
     
     private final Map<String, DataSource> dataSourceMap = new HashMap<>();
     
-    private final FileConfiguration configuration = new FileConfiguration("seata.conf");
+    private final String applicationId;
+    
+    private final String transactionServiceGroup;
+    
+    private final boolean enableSeataAT;
+    
+    public SeataATShardingTransactionManager() {
+        FileConfiguration configuration = new FileConfiguration("seata.conf");
+        enableSeataAT = configuration.getBoolean("sharding.transaction.seata.at.enable", true);
+        applicationId = configuration.getConfig("client.application.id");
+        transactionServiceGroup = configuration.getConfig("client.transaction.service.group", "default");
+    }
     
     @Override
     public void init(final DatabaseType databaseType, final Collection<ResourceDataSource> resourceDataSources) {
-        initSeataRPCClient();
-        for (ResourceDataSource each : resourceDataSources) {
-            dataSourceMap.put(each.getOriginalName(), new DataSourceProxy(each.getDataSource()));
+        if (enableSeataAT) {
```

Comment:
If turn off this flag, seata client is not initialized and seat-at is disabled?

Reply:
yes it is，meantime physical datasource will not be wrapped to seata DataSourceProxy